packages/core/src/execute.js

```diff
@@ -10,18 +10,15 @@ const prepareRequest = require('./http-middlewares/before/prepare-request');
 const constants = require('./constants');
 
 const executeHttpRequest = (input, options) => {
-  options = _.extend(
-    {},
+  options = {
```

Comment:
I found after `_.extend(a, b)`, the symbol keys in the object are gone, so I've replaced `_.extend()` with object spread syntax (`{...a, ...b}`).